src/main/java/net/sf/jabref/collab/EntryChange.java

```diff
@@ -85,7 +85,7 @@ public boolean makeChange(BasePanel panel, BibDatabase secondary, NamedCompound
         boolean allAccepted = true;
 
         Enumeration<Change> e = children();
-        for (; e.hasMoreElements();) {
+        while (e.hasMoreElements()) {
```

Comment:
Didn't know that one can have a foreach loop for Enumerations (there are these iterator-whiles everywhere in the code), but now I googled and it wasn't that hard...